style.css

```diff
@@ -1126,77 +956,44 @@ i.close.show {
     display: none;
   }
 
-  #card3 h5 {
-    width: 295px;
-    height: 81px;
-    font-family: "Crete Round", sans-serif;
-    font-weight: 400;
-    font-size: 32px;
-    line-height: 44px;
-    color: #fff;
-    flex: none;
-    order: 0;
-    flex-grow: 0;
-    margin-top: 80px;
-  }
-
-  #card3 p {
-    width: 286px;
-    height: 99px;
-    font-family: "Inter", sans-serif;
-    line-height: 25px;
-    color: #fff;
-    flex: none;
-    order: 1;
-    flex-grow: 0;
-  }
-
-  #card3 ul {
+  .devider2 {
     display: flex;
     flex-direction: row;
-    gap: 12px;
-    width: 287px;
-    height: 40px;
-    flex: none;
-    order: 2;
-    flex-grow: 0;
-    font-family: "Inter", sans-serif;
-    color: #fff;
-  }
-
-  article#card1 h5 {
-    display: none;
-  }
-
-  article#card1 p {
-    display: none;
-  }
-
-  article#card1 ul {
-    display: none;
-  }
-
-  article#card2 h5 {
-    margin-top: 137px;
+    align-items: center;
+    position: relative;
+    background: #1c1a19;
+    justify-content: space-around;
+    width: 100%;
   }
 
-  button.card2-button {
-    display: none;
+  .middle {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
   }
 
-  article#card3 h5 {
+  .center {
     display: flex;
+    flex-direction: column;
+    justify-content: space-between;
     align-items: center;
-    margin-top: 137px;
+    padding: 20px;
   }
 
-  .devider2 {
+  .buttonGroup {
+    gap: 10px;
+  }
+
+  .buttonGroup_button {
     display: flex;
-    flex-direction: row;
     align-items: center;
-    position: relative;
-    background: #1c1a19;
-    justify-content: space-around;
-    width: 100%;
+    justify-content: space-between;
+    gap: 10px;
+    width: 140px;
+    color: #fff;
+    background: #ff6b00;
+    border: none;
+    padding: 12px;
+    font-size: 15px;
+    cursor: pointer;
   }
```

Comment:
- [ ] Please make sure to follow the Figma design for the different states of the button here as I've noticed you forgot to implement the `:hover` and `:active` states.
- [ ] Here's the design as a reminder 😉
  ![firefox_IJ0QIflmz9](https://user-images.githubusercontent.com/104968236/233354667-b29dcf5d-c8ce-4d88-8c57-654067f20305.png)